pkg/eval/builtin_special.go

```diff
@@ -215,6 +218,7 @@ func (op fnWrap) invoke(fm *Frame) error {
 // UseForm = 'use' StringPrimary
 func compileUse(cp *compiler, fn *parse.Form) effectOpBody {
 	var name, spec string
+	var name_arg *parse.Compound
```

Comment:
Go variables never use snake_case. Also, the code below does not need to know that the name argument is kept in a `parse.Compound` node, so declare it with the more general `parse.Node`:

```go
var nameArg parse.Node
```

Reply:
That results in these build errors:

```
pkg/eval/builtin_special.go:233:11: cannot use fn.Args[0] (type *parse.Compound) as type *parse.Node in assignment:
        *parse.Node is pointer to interface, not interface
pkg/eval/builtin_special.go:240:11: cannot use fn.Args[1] (type *parse.Compound) as type *parse.Node in assignment:
        *parse.Node is pointer to interface, not interface
pkg/eval/builtin_special.go:249:21: nameArg.Range undefined (type *parse.Node is pointer to interface, not interface)
```

Reply:
You wrote *parse.Node I assume? Use parse.Node.

Reply:
Doh! Thx.